x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -320,15 +320,19 @@ export class Plugin implements ISecuritySolutionPlugin {
 
     const securityRuleTypeWrapper = createSecurityRuleTypeWrapper(securityRuleTypeOptions);
 
-    plugins.alerting.registerType(securityRuleTypeWrapper(createEqlAlertType(ruleOptions)));
+    plugins.alerting.registerType(
+      securityRuleTypeWrapper(createEqlAlertType({ ...ruleOptions, ...ruleAdditionalOptions }))
+    );
     if (!experimentalFeatures.esqlRulesDisabled) {
-      plugins.alerting.registerType(securityRuleTypeWrapper(createEsqlAlertType(ruleOptions)));
+      plugins.alerting.registerType(
+        securityRuleTypeWrapper(createEsqlAlertType({ ...ruleOptions, ...ruleAdditionalOptions }))
+      );
```

Comment:
should maybe these type registrations hidden behind the new feature flag as well?

Reply:
We might, but not necessarily need to. Since the rules will not contain response_actions defined, the executors of there registered rules will not make any action.

Reply:
thanks for the explanation!